PyProfane/functions.py

```diff
@@ -67,6 +79,7 @@ def censorSentences(comments: List[str]) -> List[str]:
     '''
 
     sentences = []
+    comments = [replaceSubstitutes(comment) for comment in comments]
```

Comment:
@RimorRes There might be a certain possibility of a fault here because it will replace 0's that are actually meant to be 0. Could you take there that this kind of stuff is handled within `censorWord()` without changing the actual sentences?

Reply:
For example '10$' might get changed to 'ios' even though that's not supposed to happen, this should only be as a check. Thanks very much!